doc/preface.chapter.md

```diff
@@ -42,7 +42,7 @@ shows the status of tests for the `nixpkgs-unstable` channel.
 
 The tests are conducted by a cluster called [Hydra](https://nixos.org/hydra/),
 which also builds binary packages from the Nix expressions in Nixpkgs for
-`x86_64-linux`, `i686-linux` and `x86_64-darwin`.
+`x86_64-linux`, `i686-linux` and `aarch64-darwin`.
```

Comment:
I don't think this is related to the change in https://github.com/NixOS/nixpkgs/pull/361521 which is just for building cross toolchains. This should probably mention all 4 platforms since we still build and cache `x86_64-darwin`.

Reply:
It is entirely possible I misunderstand the docs (wouldn't be the first time).

How do you read this paragraph in context?

I read it as: these are the platforms for which we build and cache pkgsCross packages.

Reply:
I don't see any mention of "cross" on the chapter except "cross-reference" which I would read as:

> Cross-reference: An instance within a document which refers to related information elsewhere in the same document.